### ISSUE TITLE
Copy and paste text contains extra characters like percents (%) etc. in PDF exported from Word

### ISSUE THREAD

**Issue by ChrisCinelli** [coolwanglu/pdf2htmlEX#623]
If you export a PDF from word and try to copy and paste the text you get tons of extra characters. In particular, spaces becomes `%`, and others characters are displayed as question marks or squares.

PDF and generated HTML are included.
[pdf.zip](https://github.com/coolwanglu/pdf2htmlEX/files/178724/pdf.zip)

**Comment by ChrisCinelli** [comment]
I tried `--space-as-offset 1`, `--correct-text-visibility 1`, `--process-nontext 1`, `--tounicode -1` and `--tounicode 1` and none of them fix it.

**Comment by coolwanglu** [comment]
There must be an earlier bug for this, but I couldn't find it.

The problem is there's nowhere in the font file nor in the ToUnicode File that marks this glyph as a whitespace. The only thing we know is that the glyph is empty. 

One thing I can do is to create a flag something like `--empty-glyph-to-offset`.

**Comment by ChrisCinelli** [comment]
How will it work ? 
Does it check if the glyph is empty in the font and later a space is used in the HTML?

Do you have any idea if this is the same way how the Chrome's PDF viewer does it?

**Comment by coolwanglu** [comment]
Yes, if there's nothing in the glyph, it will be treated as an offset shift, not necessary but often a whitespace.

Not sure about PDF viewers, but I guess so.
